src/researches/functionWordsInKeyphrase.js

```diff
@@ -0,0 +1,30 @@
+import getWords from "../stringProcessing/getWords";
+import getLanguage from "../helpers/getLanguage";
+import getFunctionWordsFactory from "../helpers/getFunctionWords.js";
+const getFunctionWords = getFunctionWordsFactory();
+import { filter, get, includes, isEmpty } from "lodash-es";
```

Comment:
Please separate the imports from the rest. And preferably also first the external imports and then the internal imports.